test/performance/README.md

```diff
@@ -49,3 +49,9 @@ If the link above do not work you should be able to find it on the ENTUR web:
 #### Germany
 
 - [Tidied GTFS data](https://otp-performance.leonard.io/data/germany/germany-2022-08-23.tidy.gtfs.zip)
+
+#### Skånetrafiken
+- [Sweden NeTEx](https://ressapublic.blob.core.windows.net/ressapub/ST_netex.zip)
```

Comment:
Since these URLs are now easily visible in the build confg, we could leave them out here and replace them with a link. This is less to maintain. WDYT?

If you agree could you do the same for the other locations?

Reply:
I've changed the documentation now